backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -208,16 +218,12 @@ func (c *ClientManager) Close() {
 	c.db.Close()
 }
 
-func initDBClient(initConnectionTimeout time.Duration) *storage.DB {
-	driverName := common.GetStringConfig("DBConfig.DriverName")
-	var arg string
-
-	switch driverName {
-	case "mysql":
-		arg = initMysql(driverName, initConnectionTimeout)
-	default:
-		glog.Fatalf("Driver %v is not supported", driverName)
-	}
+func InitDBClient(initConnectionTimeout time.Duration) *storage.DB {
+	// Allowed driverName values:
+	// 1) To use MySQL, use `mysql`
+	// 2) To use PostgreSQL, use `pgx`
```

Comment:
Why do we choose `pgx` for postgresql in `DBDriverName` flag? Is there an alternative which is easier for people to understand?

Reply:
+1, `pgx` feels weird, albeit I saw that's the name of the go package you're using. I think `postgres` would be still better here.

Reply:
I used this because we are using the package `pgx` as the PostgreSQL driver, and it is needed in the `sql.Open()` function to create the correct DB instance. See the example [here](https://pkg.go.dev/github.com/jackc/pgx/stdlib). I agree it may be quite confusing to people, but I was weighting this against being consistent with the driver name we are using, so I eventually opted for making it clear in the comments to reduce the confusion. WDYT?

Reply:
Okay I think @Linchin 's point is reasonable, we just need to make sure we document this parameter is being used as  a sql driver flag: https://github.com/golang/go/wiki/SQLDrivers.